contrib/storage-hbase/src/main/java/org/apache/drill/exec/store/hbase/HBaseUtils.java

```diff
@@ -139,4 +142,26 @@ public static Filter orFilterAtIndex(Filter currentFilter, int index, Filter new
     return Bytes.compareTo(left, right) < 0 ? left : right;
   }
 
+
+  /**
+   * Verify the presence of a column family in the schema path of the hbase table or whether the schema path is
+   * the row key column.
+   *
+   * @param columns List of the selected columns
+   * @param hTableDesc HTableDescriptor of HBase/MapR-DB_binary table (consists the details about that table)
+   * @throws DrillRuntimeException if column family does not exist, or is not row_key column.
+   */
+  public static void verifyColumns(List<SchemaPath> columns, HTableDescriptor hTableDesc) {
```

Comment:
As it turns out, this PR is introducing the very ambiguity that DRILL-5830 tried to remove from DRILL-5546. That is, we now have to separate ways to expand the wildcard: this way and the project push down way. We need this commit, so we'll accept this for now. But, moving forward, we should clear up this ambiguity.

Reply:
DRILL-5546 introduced expanding wildcard in the `verifyColumns()` method with renaming to `verifyColumnsAndConvertStar()`. But this PR keeps the original `verifyColumns()` method and the same in DRILL-5830 (the star isn't expanded here). Just the method have started to be common to HBase and MapR-DB. 
And MapR-DB plugin for now leverages the same way of project push down as HBase due to the `MapRDBBinaryTable` introducing.
Or did I miss something?